CMakeLists.txt

```diff
@@ -72,8 +72,15 @@ macro(build_benchmark)
   )
 endmacro()
 
-if(NOT benchmark_FOUND)
+if(NOT benchmark_FOUND OR "${benchmark_VERSION}" VERSION_LESS 1.5.0)
   build_benchmark()
+elseif(benchmark_FOUND)
+  # Ubuntu Focal has a packaging bug where libbenchmark_main has no symbols,
+  # causing linker failures. I'm pretty sure it shouldn't be a static library.
```

Comment:
@cottsay mind to leave a link to the bug report upstream and TODO to drop this if/once that one is resolved?

Reply:
I'll add a link to the bug.
I'm hesitant to drop the special case until the problem doesn't exist at all on the supported platforms. Since we can't express a minimum version on a rosdep rule, someone could conceivably not update the package, and I think we should still do the right thing here. Maybe we can drop in H-turtle?

Reply:
Sounds good. At any rate, I don't mind when we drop this as long as our future selves understand why it's there.

Reply:
Absolutely, well put.

Reply:
1473804